Vic2ToHoI4Tests/HoI4WorldTests/Hoi4CountryTests.cpp

```diff
@@ -1,40 +1,41 @@
-//#include "Mocks/CountryMapperMock.h"
-//#include "Mocks/GovermentMapperMock.h"
-//#include "Mocks/GraphicsMapperMock.h"
-//#include "Mocks/Hoi4StateMock.h"
-//#include "Mocks/Vic2CountryMock.h"
-//#include "Mocks/Vic2WorldMock.h"
+//#include "Color.h"
```

Comment:
Why is all this commented out?

Reply:
If you peer inside, you'll see the abyss looking back at you.

Reply:
Still too many dependencies on data files that I don't have test files for. Eliminating the mocks was good but completely fixing these tests is out of scope for now.